examples/MQ/serialization/sampler.cxx

```diff
@@ -64,6 +66,9 @@ struct Sampler : fair::mq::Device
             ExHeader header;
             header.eventNumber = idx;
 
+            // temporary data check before transfer
```

Comment:
Maybe change this a bit?

```suggestion
            // Temporarily apply data processing steps here to check that they work out.
```

Reply:
Alright. Modified to something like you suggest.